src/backend/src/utils/users.utils.ts

```diff
@@ -52,3 +53,23 @@ export const rankUserRole = (role: Role) => {
       return 1;
   }
 };
+
+/**
+ * Produce a array of User with given userIds
+ * @param userIds array of userIds as an array of integers
+ * @returns array of User with type = User[]
```

Comment:
instead of saying "with type = user[]" here, add a type annotation to the function itself of `User[]`